tests/network/service_mesh/test_service_mesh.py

```diff
@@ -56,9 +57,13 @@ def test_authentication_policy_from_mesh(
         vm_fedora_with_service_mesh_annotation,
         httpbin_service_service_mesh,
     ):
-        assert_authentication_request(
+        request_response = authentication_request(
```

Comment:
The `authentication_request`  is implicitly asserting already because it expects the output as input.
The underling `verify_console_command_output` helper does it, causing all its users to use it badly in tests.

This is what I would expect to see here:
```python
result = exec_console_command(vm, authentication_command())
assert EXPECTED_MESH_SUCCESS_OUTPUT in result
```

`exec_console_command` should handle EOF and TIMEOUT cases.
It is expected to return the `child.before.decode('utf-8')` on success (or failure) to reach the prompt.

Reply:
Ack.